docs/howto/use-with-samba.md

```diff
@@ -0,0 +1,239 @@
+# Using authd with Samba
+
+The UIDs and GIDs assigned to users and groups by authd are unique to each
+machine. This means that when using authd with Samba, the UIDs and GIDs of users
+and groups on the Samba server will not match those on the client machines,
+which leads to permission issues.
+
+To avoid these issues, you can use Samba with ID mapping. This ensures that the
+UIDs and GIDs of users and groups are mapped correctly across all machines.
+
+## Setting up Samba with ID Mapping
+
+This guide will walk you through setting up a Samba server with ID mapping. The
+user `alice` will be able to access a shared directory on the server from a
+client machine.
+
+---
+
+### Steps for the Server
+
+1. **Install Samba:**
+   Install the Samba server package:
+
+   ```bash
+   sudo apt update
+   sudo apt install samba
+   ```
+
+2. **Create the Shared Directory:**
+   Create the directory to be shared and set ownership to the `alice` user:
+
+   ```bash
+   sudo mkdir -p /srv/samba/alice
+   sudo chown alice:alice /srv/samba/alice
+   ```
+
+3. **Edit Samba Configuration:**
+   Open the Samba configuration file:
+
+   ```bash
+   sudo editor /etc/samba/smb.conf
+   ```
+
+   Add the following section at the end of the file:
+
+   ```ini
+   [alice]
+   path = /srv/samba/alice
+   browsable = yes
+   writable = yes
+   valid users = alice
+   ```
+
+   - **Explanation:** This section defines a Samba share named `alice` located
+   at `/srv/samba/alice`. It is visible to users on the network (`browsable`),
+   allows writing (`writable`), and restricts access to the `alice` user (`valid
+   users`).
+
+4. **Create a Samba User for `alice`:**
+   Add the `alice` user to the Samba database and set a password:
+
+   ```bash
+   sudo smbpasswd -a alice
+   ```
+
+   Follow the prompts to set the Samba password for the user.
+
+5. **Restart Samba Service:**
+   Restart the Samba service to apply the changes:
+
+   ```bash
+   sudo systemctl restart smbd
+   ```
+
+---
+
+### Steps for the Client
+
+1. **Install Samba Client:**
+   Install the required packages for connecting to Samba shares:
+
+   ```bash
+   sudo apt update
+   sudo apt install smbclient cifs-utils
+   ```
+
+2. **Test Access to the Share:**
+   Test connectivity using `smbclient` (replace `$SERVER` with the Samba
+   server's hostname or IP address):
+
+   ```bash
+   smbclient //$SERVER/alice -U alice
+   ```
+
+   Enter the Samba password for `alice` when prompted. If successful, a `smb: \>`
+   prompt appears.
+
+3. **Mount the Share:**
+   Create a mount point for the share:
+
+   ```bash
+   mkdir -p /home/alice/samba
+   ```
+
+   Mount the share using the `cifs` filesystem type:
+
+   ```bash
+   sudo mount -t cifs //$SERVER/alice /home/alice/samba -o user=alice,uid=$(id -u alice),gid=$(id -g alice)
+   ```
+
+   Enter the Samba password for `alice` when prompted.
+
+4. **Optional: Add to `/etc/fstab` for Persistent Mounting:**
```

Comment:
```suggestion
4. **Optional: Add to `/etc/fstab` for persistent mounting:**
```

Title could be more explicit: add _what_ to `/etc/fstab`?

Reply:
Changed to 

```
**Optional: Add the share to `/etc/fstab` for persistent mounting:**
```